docs/request-for-comment/zz-RFC-template.md

```diff
@@ -0,0 +1,26 @@
+# RFC: _title_
+_The title is what you want comments on. Use the active voice, and future tense. Example: The website will be built using the React framework_
+* Date: _YYYY-MM-DD_
+* Authors: _tag yourself and other authors here_
+* Status: **PROPOSED**
+* Related Issues: _add links to github/zenhub issues_
+* Related Discussions: _add links to other documents, slack discussions, etc_
+
```

Comment:
add link to the RFC PR

Reply:
We want to add a link to itself or am I missing something here?